src/controllers/actionsteps.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable import/extensions */
+const logger = require("../middleware/logger");
+const ActionStep = require("../models/actionSteps");
+const asyncHandler = require("../middleware/async");
+
+const getactionSteps = asyncHandler(async (req, res, next) => {
+  try {
```

Comment:
The point of the `asyncHandler` middleware is to avoid repeating the `try...catch` idiom as it's already handled in the asyncHandler middleware.

Reply:
fixed.